### PR TITLE
Fix missing hs_version.h header (closes #198)

### LINKED ISSUE #198: Install doesn't include hs_version.h
After compiling and installing vectorscan locally, the hs_version.h is missing in the installed header files:

```
/work/hyperscan-java-native/cppbuild/include/hs/hs.h:42:10: fatal error: hs_version.h: No such file or directory
   42 | #include "hs_version.h"
```

```
[root@e2de98379481 include]# find .
.
./hs
./hs/hs_common.h
./hs/hs_runtime.h
./hs/hs.h
./hs/hs_compile.h
```

It seems to me that this is related to the change making sure that the version numbers don't get out of sync:
https://github.com/VectorCamp/vectorscan/commit/16604f9539929b5d19dc81d529c36770fd8eebfd

Now hs_version.h is suddenly part of the required development headers, which was not the case before.

### PR DESCRIPTION
**Pull request by gliwka**
This PR adds the missing hs_version.h (issue #198) to the files to be installed

### CODE REVIEW

**Comment by markos** [comment]
Would you mind sending this PR against develop branch? It doesn't matter much in this particular case as they are exactly the same.

**Comment by gliwka** [comment]
Will do, just need to rebase 👍

**Comment by gliwka** [comment]
@markos Done :-)

File: CMakeLists.txt [hunk 1]
```diff
@@ -225,6 +225,7 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${EXTRA_CXX_FLAGS}")
 endif()
 
 SET(hs_HEADERS
+    ${PROJECT_BINARY_DIR}/hs_version.h
```
Comment (markos) [inline CMakeLists.txt:228]: LGTM, Approved

**Review by markos — APPROVED** [review]
LGTM, Approved

**Pull request merged**